.github/workflows/cla.yml

```
@@ -1,12 +1,10 @@
 name: 'CLA Assistant'
 on:
-  issue_comment:
-    types: [created]
-  pull_request_target:
-    types: [opened, closed, synchronize]
+  workflow_dispatch:
 
 jobs:
   CLAssistant:
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)